tests/irdl/test_declarative_assembly_format.py

```diff
@@ -1710,23 +1710,22 @@ class ParamOne(ParametrizedAttribute, TypeAttribute, Generic[_T]):
         name = "test.param_one"
         p: ParameterDef[_T]
 
-        @classmethod
+        @staticmethod
```

Comment:
I think this looks good, much closer to what I imagined

Reply:
Is there any hope to have all `ParametrisedAttribute`s define this? perhaps with a method defined in `@irdl_attr_definition`? (Perhaps not as this PR)